crates/pop-contracts/src/call/metadata.rs

```diff
@@ -68,6 +81,39 @@ fn process_args(message_params: &[MessageParamSpec<PortableForm>]) -> Vec<Param>
 	args
 }
 
+/// Generates a list of processed argument values for a specified contract message,
+/// wrapping each value in `Some(...)` or replacing it with `None` if the argument is optional
+///
+/// # Arguments
+/// * `path` -  Location path of the project.
+/// * `message_label` - Label of the contract message to retrieve.
+/// * `args` - Argument values provided by the user.
+pub fn generate_message_args(
```

Comment:
```suggestion
pub fn process_message_args(
```
We are not generating args, we are processing those provided.

Reply:
That's true. Bad naming